core/src/operations/mod.rs

```diff
@@ -32,6 +32,10 @@ pub enum Operation {
     /// SYSCALL. Thus, this operation can be executed only inside a SYSCALL code block.
     Caller,
 
+    /// Pushes the current value of the cycle counter onto the stack. This operation is used to
+    /// measure the execution time of a program.
+    Clk,
+
```

Comment:
nit: this comment doesn't feel entirely accurate, since we're not actually measuring time, and we don't really talk about this as a "cycle counter" anywhere else. I'd probably amend it to something like:

> Pushes the value of the current clock cycle onto the stack. This operation can be used to measure the number of cycles it has taken to execute the program up to the current instruction.

Reply:
yeah- it could be a proxy but not an accurate description. I've updated it to what you had suggested.